apps/challenge-registry/app/src/app/app-sections.ts

```diff
@@ -3,7 +3,7 @@ import { NavbarSection } from '@sagebionetworks/challenge-registry/ui';
 export const APP_SECTIONS: { [key: string]: NavbarSection } = {
   about: {
     name: 'About',
-    summary: 'About the Challenge Registry',
+    summary: 'About OpenChallenges',
```

Comment:
English question: Do we need article here?
General question: Is our app called "openChallenges" or "Open Challenges"? - that's confused me a bit, lol.

Reply:
The official name based on the feedback received is "OpenChallenges" (no space, PascalCase).